pyfesom/regriding.py

```diff
@@ -9,6 +9,7 @@
 from scipy.spatial import cKDTree
 import numpy as np
 from collections import namedtuple
+from .load_mesh_data import fesom2depth
```

Comment:
You see. this line appears here again, while we already merge PR with this change. This is why it's a good idea to do PRs with branches.